File: Check for subsequence/Check for subsequence.cpp
#include <iostream>
#include <string>
class Solution{
    public:
    bool isSubSequence(string A, string B) 
    {
        int l=0;
        for(int i=0;i<B.size();i++){
            if(B[i]==A[l]){
                l++;}
        }
        if(l==A.size()){
            return true;}
            
        else {
            return false;}
            
    }
    
};

File: Check for subsequence/Check for subsequence.py
class Solution:
    def isSubSequence(self, A, B):
        j = 0
        for i in range(len(B)):
            if j < len(A):
                if B[i] == A[j]:
                    j += 1
        if j == len(A):
            return True
        else:
            return False